lib/libgcc_eh/Makefile.inc

```diff
@@ -1,11 +1,12 @@
 # $FreeBSD$
 
 COMPILERRTDIR=	${SRCTOP}/contrib/compiler-rt
-.if defined(LIBCHERI)
-# XXXAR: we should do this for MIPS too
+.if ${MK_CHERI} != "no"
+WANT_CHERI?=	hybrid
 UNWINDINCDIR=   ${SRCTOP}/contrib/cheri-libunwind/include
 UNWINDSRCDIR=   ${SRCTOP}/contrib/cheri-libunwind/src
 .else
+WANT_CHERI=hybrid
```

Comment:
This lines in this branch of the `.if` statement are evaluated when `${MK_CHERI} == "no"` and don't include cheri-libwunwind paths so this still doesn't make sense.  It made some sense before when it was `.if defined(LIBCHERI)`